style.css

```diff
@@ -553,69 +556,302 @@ textarea {
   font-size: 22px;
 }
 
-@media (max-width: 768px) {
-  .card-body {
-    overflow-x: hidden;
+@media (min-width: 768px) {
+  .first {
+    display: flex;
+    justify-content: center;
+    width: 100vw;
+    min-height: 8vh;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: transparent;
   }
 
-  .display-centered {
+  .first-img {
     display: none;
   }
 
-  .childs p {
-    padding-left: 20px;
-    font-size: 16px;
+  .second {
+    background-image: url(./images/Header-llustration-desktop.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    gap: 0;
+    width: 100vw;
+    height: 100vh;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] I have noticed that the background image may be one of the causes of the Horizontal scroll 😞 . The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling 😞 . As the previous reviewer suggested. Kindly ensure that there is no horizontal scroll in the entire application screen size. 👍 

- Consider the screenshot for reference. 😎 
![horizontal scroll](https://user-images.githubusercontent.com/99394374/213892770-3f7ebfb4-8cd6-4306-a122-c85cb66edf47.PNG)